### DISCUSSION TITLE
Cura settings

### DISCUSSION THREAD

**Discussion by OB1-OB2** [Jyers/Marlin#288] [Q&A] [answered]
Hey all, 

I've been using Prusaslicer with no issues with the FW and all is working really well.

I wanted to print something with tree supports so I booted up Cura and am using the default settings for a 0.6mm nozzle and PLA at 215C. But with Cura, it just prints nothing but a glob and looks like the extruder is frankly having a very bad case of stomach issues.

The exact same print from Prusaslicer comes out well except there's no tree supports so breaking the supports is a problem.

I'm a bit of noob, so can someone please point towards what to look at in Cura settings to solve this?

Any help is much appreciated. Thanks.

**Comment by Cal-LF** [comment, marked as answer]
Your question may be better answered on a Cura forum.

**Reply by OB1-OB2** [reply]
Good point. Let me rephrase, using this FW, do I have to do something special in Cura relative to using Creality FW?
If the answer is no, I'll take it over to Cura forum but I'm willing to bet people there will tell me it's a FW issue.

**Reply by marcomania75** [reply]
i have no issues with Cura and E3V2-UBL-BLTouch-10x10-v4.2.2.bin and I have no special settings related to this firmware

**Reply by OB1-OB2** [reply]
Thank you. I'm off to the cura forum.

**Comment by Jyers** [comment]
Just interested, is this anything like #272?

**Reply by OB1-OB2** [reply]
It was a noob user error problem. All's well with the FW, which BTW, is a good 1000x better than the junkie stock FW that comes with the printer. Thanks for your efforts here.
